gourmet/reccard.py

```diff
@@ -2526,9 +2526,9 @@ def grab_selection (model, path, iter, args):
         strings=[]
         iters=[]
         tv.get_selection().selected_foreach(grab_selection,(strings,iters))
-        str=string.join(strings,"\n")
-        selection.set('text/plain',0,str)
-        selection.set('STRING',0,str)
+        _str="\n".join(strings)
```

Comment:
That was a bad name.

Reply:
Since it's changing, it would make sense to use a more descriptive name.

Reply:
like `ingredients`?

Reply:
Fully agree. Unfortunately, it's a bit difficult to guess what's inside this variable, but now I know :smiley: 
Done